### ISSUE TITLE
LDP-RS creation with POST and Turtle payload results in a LDP-NR.

### ISSUE THREAD

**Issue by scossu** [scossu/lakesuperior#56]
### Environment

Operating system: various

Python version: N/A

LAKEsuperior release, branch, or commit #: 1.0.0a13

### Observed behavior

(from @acoburn 's email:)

[...] issue that I ran into related to trying to create a direct container via POST (PUT seems to work correctly). This is the file I'm using:

```
PREFIX dcterms: <http://purl.org/dc/terms/>
PREFIX ldp: <http://www.w3.org/ns/ldp#>

<> dcterms:title "Direct Container" ;
    ldp:membershipResource <http://localhost:8000/ldp/member> ;
    ldp:hasMemberRelation dcterms:relation .
```

(assume the pre-existing presence of /ldp/member)

Whenever I try to create a new resource via POST, the new resource is stored as an LDP-NR. Here are the various commands I've used:

```
curl -i localhost:8000/ldp -XPOST --data-binary @dc.ttl
curl -i localhost:8000/ldp -XPOST --data-binary @dc.ttl -H"Content-Type: text/turtle"
curl -i localhost:8000/ldp -XPOST --data-binary @dc.ttl -H"Content-Type: text/turtle" -H"Slug: dc"
curl -i localhost:8000/ldp -XPOST --data-binary @dc.ttl -H"Content-Type: text/turtle" -H"Slug: dc" -H"Link: <http://www.w3.org/ns/ldp#DirectContainer>; rel=\"type\""
```

In every case, the new resource is an LDP-NR. Though when I use equivalent commands to create a resource via PUT, everything works correctly. This same issue seems to apply to regular RDF source/container resources as well: they are created correctly as an LDP-RS under PUT but not under POST.

### Expected behavior

Containers should be created with POST the same way they are created with PUT.

### Other notes worth mentioning

This may be the case with indirect containers as well. Needs testing.

**Comment by scossu** [comment]
@acoburn This is not restricted to LDP-DCs. Changing issue title.

**Comment by scossu** [comment]
Fixed in #59.
